runconfig/parse.go

```diff
@@ -187,6 +188,13 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 			MemorySwap = parsedMemorySwap
 		}
 	}
+	if *flSwappiness < 0 || *flSwappiness > 100 {
+		return nil, nil, cmd, fmt.Errorf("invalid value:%d. valid memory swappiness range is 0-100", *flSwappiness)
+	}
+	// This is to ensure that we do not try to override swappiness value for default case
+	if *flSwappiness == 60 {
+		*flSwappiness = -1
+	}
```

Comment:
@hqhq @lizf-os Yes Agree.. there is a gap in handling the special case you mentioned. I have been thinking over how to

case1: handle if somebody tries to set swappiness = 60 explicitly
case2: nobody asks us to configure swappiness
In case1 we should try to write the value.  (case2 implicitly swappiness value should be set to -1)
so that we ignore writing any value.